cmd/kubeadm/app/cmd/upgrade/plan_test.go

```diff
@@ -62,16 +62,27 @@ func TestSortedSliceFromStringIntMap(t *testing.T) {
 func TestPrintAvailableUpgrades(t *testing.T) {
 	featureGates := make(map[string]bool)
 	var tests = []struct {
+		name          string
 		upgrades      []upgrade.Upgrade
 		buf           *bytes.Buffer
 		expectedBytes []byte
+		externalEtcd  bool
 	}{
 		{
+			name:     "Up to date",
 			upgrades: []upgrade.Upgrade{},
 			expectedBytes: []byte(`Awesome, you're up-to-date! Enjoy!
```

Comment:
huh, i wonder how the old code passed `go fmt`. it complains for me when not indenting the members property.